src/ImageSharp/Formats/Ani/AniFrameMetadata.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+using SixLabors.ImageSharp.Metadata;
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace SixLabors.ImageSharp.Formats.Ani;
+
+/// <summary>
+/// Provides Ani specific metadata information for the image.
+/// </summary>
+public class AniFrameMetadata : IFormatFrameMetadata<AniFrameMetadata>
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AniFrameMetadata"/> class.
+    /// </summary>
+    public AniFrameMetadata()
+    {
+    }
+
+    /// <summary>
+    /// Gets or sets the display time for this frame (in 1/60 seconds)
+    /// </summary>
+    public uint Rate { get; set; }
+
+    /// <summary>
+    /// Gets or sets the frames count of **one** "icon" chunk.
+    /// </summary>
+    public int FrameCount { get; set; } = 1;
+
+    /// <summary>
+    /// Gets or sets the encoding width. <br />
+    /// Can be any number between 0 and 255. Value 0 means a frame height of 256 pixels or greater.
+    /// </summary>
+    public byte? EncodingWidth { get; set; }
+
+    /// <summary>
+    /// Gets or sets the encoding height. <br />
+    /// Can be any number between 0 and 255. Value 0 means a frame height of 256 pixels or greater.
+    /// </summary>
+    public byte? EncodingHeight { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="ImageMetadata"/> of one "icon" chunk.
+    /// </summary>
+    public ImageMetadata? SubImageMetadata { get; set; }
```

Comment:
We shouldn't nest sub data like this. The metadata needs to own all required data.